### PR TITLE
[custom_dc] fold in IITM updates

### PR DESCRIPTION
**Pull request by beets**
Ports most functional changes from #1529, #1758 and [iitm branch](https://github.com/datacommonsorg/website/tree/iitm) (iitm branch is now behind this PR).

Includes:
- All html templates updated to use custom base template
- Data download tool added to header
- Primary button color overwritten
- Customized landing pages for place, browser, map, scatter, timeline, download tools
- Loads JS libraries from CDN's where possible (except particle.js)
- Load BQ SQL button

There are some minor differences from datacommons.iitm.ac.in
- Fonts don't look as crisp anymore (especially bold text on the tool info pages). Not sure if this is because of font-awesome
- Slight spacing updates

Before (left, prod), after (right, localhost):

image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Review by SenthamizhanV — APPROVED** [review]
Pulled this branch and deployed the website locally. Everything works fine. I could not view the /place page since I do not have the DC API key. All other pages look good.

The difference in fonts can be ignored. One place I noticed there is a difference in the text below 'India Data Commons' in the homepage banner. The font-weight is different - this PR has value of 600 but the prod version has 500. But we can leave it at that.

File: server/config/topic_page_pb2.py [hunk 1]
```diff
@@ -2,9 +2,10 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
```
Comment (shifucun) [inline server/config/topic_page_pb2.py:2]: nit: can you check the protoc version with `protoc --version`. The current file is generated using 3.21.9 and I suspect this change is from using an older version.

File: server/configmodule.py [hunk 2]
```diff
@@ -87,9 +87,10 @@ class TidalConfig(CustomConfig):
 
 
 class IitmConfig(Config):
-    IITM = True
     NAME = "IITM"
```
Comment (shifucun) [inline server/configmodule.py:90]: nice!

File: server/routes/static.py [hunk 3]
```diff
@@ -42,13 +40,20 @@ def about():
     env_name = current_app.config.get('ENV_NAME', None)
```
Comment (shifucun) [inline server/routes/static.py:40]: move `env_name = current_app.config.get('ENV_NAME', None)` to the top of this module and only declare once?

File: server/routes/tools.py [hunk 4]
```diff
@@ -76,7 +104,10 @@ def stat_var():
 
 @bp.route('/download')
 def download():
+    env_name = current_app.config.get('ENV_NAME', None)
```
Comment (shifucun) [inline server/routes/tools.py:107]: same here, moved to the top of the module

File: server/config/topic_page_pb2.py (hunk 1 above)
Comment (beets) [reply server/config/topic_page_pb2.py]: Thanks for pointing that out -- reverted for now.

File: server/routes/static.py (hunk 3 above)
Comment (beets) [reply server/routes/static.py]: It requires an application context to load the value, so it won't work at the top of the module. Instead, I've added it to the request global. ptal and lmk what you think

**Review by shifucun — APPROVED** [review]

**Comment by shifucun** [comment]
Glad to see this is checked in!

File: server/routes/static.py (hunk 3 above)
Comment (shifucun) [reply server/routes/static.py]: This is concise!

**Comment by beets** [comment]
> Pulled this branch and deployed the website locally. Everything works fine. I could not view the /place page since I do not have the DC API key. All other pages look good.
> 
> The difference in fonts can be ignored. One place I noticed there is a difference in the text below 'India Data Commons' in the homepage banner. The font-weight is different - this PR has value of 600 but the prod version has 500. But we can leave it at that.

I just noticed that requesting 'Public Sans' was dropped from the header, so added it back and the text looks much better now. I've also updated the font weight for just the homepage.

**Comment by beets** [comment]
Thanks for the reviews!

@SenthamizhanV If there are other diffs you notice, we can now continue fixing this on the main branch :)

**Pull request merged**